### PR TITLE
RDS Core: IPVLAN connectivity test added

### PR DESCRIPTION
**Pull request by elenagerman**
IPVLAN connectivity test added: different nodes & same node

### CODE REVIEW

**Comment by elenagerman** [comment]
cc: @yprokule

File: tests/system-tests/rdscore/internal/rdscorecommon/ipvlan-validation.go [hunk 1]
```diff
@@ -0,0 +1,391 @@
...
+
+	return deploy
+}
+
+// VerifyIPVLANSuite container that contains tests for IPVLAN verification.
+func VerifyIPVLANSuite() {
+	Describe(
+		"IPVLAN verification",
+		Label("ipvlan-clean-cluster"), func() {
+			It("Verify IPVLAN", Label("validate-new-ipvlan-different-nodes"), reportxml.ID("75057"),
```
Comment (yprokule) [inline tests/system-tests/rdscore/internal/rdscorecommon/ipvlan-validation.go:385]: nit:
```suggestion
			It("Verify IPVLAN workloads on different nodes", Label("validate-new-ipvlan-different-nodes"), reportxml.ID("75057"),
```

File: tests/system-tests/rdscore/internal/rdscorecommon/ipvlan-validation.go [hunk 2]
```diff
@@ -0,0 +1,391 @@
...
+				VerifyIPVlanOnDifferentNodes)
+
+			It("Verify IPVLAN", Label("validate-new-ipvlan-same-node"), reportxml.ID("75562"),
```
Comment (yprokule) [inline tests/system-tests/rdscore/internal/rdscorecommon/ipvlan-validation.go:388]:
```suggestion
			It("Verify IPVLAN workload on the same node", Label("validate-new-ipvlan-same-node"), reportxml.ID("75562"),
```

File: tests/system-tests/rdscore/tests/00_validate_top_level.go [hunk 3]
```diff
@@ -26,7 +26,7 @@ var _ = Describe(
 	Label("rds-core-workflow"), func() {
 		Context("Configured Cluster", Label("clean-cluster"), func() {
 			It("Verifies SR-IOV workloads on same node and different networks",
-				Label("sriov", "sriov-same-node-different-nets"),
+				Label("sriov", "debug"),
```
Comment (yprokule) [inline tests/system-tests/rdscore/tests/00_validate_top_level.go:29]: don't forget to restore when it's ready 😸

File: tests/system-tests/rdscore/internal/rdscorecommon/ipvlan-validation.go [hunk 4]
```diff
@@ -0,0 +1,391 @@
...
+				VerifyIPVlanOnSameNode)
+		})
```
Comment (yprokule) [inline tests/system-tests/rdscore/internal/rdscorecommon/ipvlan-validation.go:390]: I'd rather move it to the `00_validate_top_level.go` file but leaving it up to U

File: tests/system-tests/rdscore/tests/00_validate_top_level.go [hunk 5]
```diff
@@ -47,6 +47,12 @@ var _ = Describe(
 			It("Verify MACVLAN", Label("macvlan", "validate-new-macvlan-same-node"), reportxml.ID("72567"),
 				rdscorecommon.VerifyMacVlanOnSameNode)
 
+			It("Verify IPVLAN", Label("ipvlan", "validate-new-ipvlan-different-nodes"), reportxml.ID("75057"),
```
Comment (yprokule) [inline tests/system-tests/rdscore/tests/00_validate_top_level.go:50]:
```suggestion
			It("Verify IPVLAN workload on different nodes", Label("ipvlan", "validate-new-ipvlan-different-nodes"), reportxml.ID("75057"),
```

File: tests/system-tests/rdscore/tests/00_validate_top_level.go [hunk 6]
```diff
@@ -47,6 +47,12 @@ var _ = Describe(
...
+				rdscorecommon.VerifyIPVlanOnDifferentNodes)
+
+			It("Verify IPVLAN", Label("ipvlan", "validate-new-ipvlan-same-node"), reportxml.ID("75562"),
```
Comment (yprokule) [inline tests/system-tests/rdscore/tests/00_validate_top_level.go:53]:
```suggestion
			It("Verify IPVLAN workloads on the same node", Label("ipvlan", "validate-new-ipvlan-same-node"), reportxml.ID("75562"),
```

File: tests/system-tests/rdscore/tests/00_validate_top_level.go (hunk 3 above)
Comment (elenagerman) [reply tests/system-tests/rdscore/tests/00_validate_top_level.go:29]: sure, sorry, leftovers

**Pull request merged**